arrow-core/src/main/kotlin/arrow/core/extensions/option/applicativeError/OptionApplicativeError.kt

```diff
@@ -211,10 +199,10 @@ fun <A> ApplicativeError<ForOption, Throwable>.catch(arg1: Function0<A>): Option
 @Deprecated(
   "@extension kinded projected functions are deprecated",
   ReplaceWith(
-  "effectCatch(arg0, arg1)",
-  "arrow.core.Option.effectCatch"
+    "Option.catch(arg0) { arg1() }",
```

Comment:
does this function exists for suspend?

Reply:
The function is `inline` so you can call `suspend` in it's lambda.